website/content/contributors/_index.md

```diff
@@ -10,51 +10,60 @@ Welcome! Are you interested in contributing to one of CNCF hosted projects? This
+If you are interested in participating in one of the programs, please check out the [CNCF mentoring repository](https://github.com/cncf/mentoring) for more details.
 
-## Community Engagement
+### Project Opportunities
 
-### Ambassadors
+The CNCF projects are always looking for new contributors. If you're interested in contributing to a CNCF project, check out the following:
 
-[Cloud Native Ambassadors](https://www.cncf.io/people/ambassadors/) (CNAs) are individuals who are passionate about [Cloud Native Computing Foundation](https://www.cncf.io/) technology and projects, recognized for their expertise, and willing to help others learn about the framework and community.
+- [CNCF Landscape](https://landscape.cncf.io/) - a map of all CNCF projects, along with their maturity level and their category
+- [CLOTributor](https://clotributor.dev/) - a tool that helps you find beginner-friendly tasks in CNCF projects
+- [Projects section below](#projects) lists all CNCF projects, with their primary language and brief information on contributing to them
 
-Successful ambassadors are people such as bloggers, influencers, evangelists who are already engaged with a CNCF project in some way including contributing to forums, online groups, community events, etc.
 
-Details on the Ambassadors program, and information on how to join CNCF as an Ambassador is available [here](https://github.com/cncf/ambassadors).
+### Technical Advisory Groups (TAGs)
 
-### Meetups
+CNCF Technical Advisory Groups (TAGs) are responsible for overseeing and coordinating the interests of end users and/or projects within a specific area of focus. These groups hold open meetings that are accessible to anyone without any obligations. You are welcome to participate in discussions or simply observe the group's activities. If you are interested in contributing to a TAG, you can find more information in their respective repositories.
 
-The Cloud Native Computing Foundation supports the worldwide community of the Cloud Native meetups. They are listed on [meetup.com](https://www.meetup.com/pro/cncf/) website.
+Although TAGs are designed to focus on specific areas, you can begin contributing through working groups (WGs) that operate under the TAGs with even more narrow scopes. WGs are community-driven groups that encourage ongoing collaboration in specific areas of interest.
 
-CNCF is currently working on expanding the Cloud Native community around the globe, and we are happy to accept the new meetup communities to join our network, and become one of the official CNCF meetups.
+More details about the CNCF TAGs is available [here](https://github.com/cncf/toc/blob/main/tags/cncf-tags.md). You can find the list of TAGs [here](https://github.com/cncf/toc/tree/main/tags).
```

Comment:
Just a FYI, it really helps people using screen readers to not name the link text "here". Instead name the link text something descriptive that explains what they will find if they follow the link.

```suggestion
Learn more about [CNCF TAGs](https://github.com/cncf/toc/blob/main/tags/cncf-tags.md) and view a [list of the CNCF TAGs](https://github.com/cncf/toc/tree/main/tags).
```